rfc/5280/index.js

```diff
@@ -0,0 +1,857 @@
+    this.key('issuerUniqueID').optional().explicit(1).bitstr(),
+    this.key('subjectUniqueID').optional().explicit(2).bitstr(),
+    this.key('extensions').optional().explicit(3).seqof(Extension)
+  );
+});
+rfc5280.TBSCertificate = TBSCertificate;
+
+// Version  ::=  INTEGER  {  v1(0), v2(1), v3(2)  }
+var Version = asn1.define('Version', function() {
+  this.int({
+    0: 'v1',
+    1: 'v2',
+    2: 'v3'
+  });
+});
+rfc5280.Version = Version;
+
+// Validity ::= SEQUENCE {
+//      notBefore      Time,
+//      notAfter       Time  }
+var Validity = asn1.define('Validity', function() {
+  this.seq().obj(
+    this.key('notBefore').use(Time),
+    this.key('notAfter').use(Time)
+  );
+});
+rfc5280.Validity = Validity;
+
+// Time ::= CHOICE {
+//      utcTime        UTCTime,
+//      generalTime    GeneralizedTime }
+var Time = asn1.define('Time', function() {
+  this.choice({
+    utcTime: this.utctime(),
+    genTime: this.gentime()
+  });
+});
+rfc5280.Time = Time;
+
+// SubjectPublicKeyInfo  ::=  SEQUENCE  {
+//      algorithm            AlgorithmIdentifier,
+//      subjectPublicKey     BIT STRING  }
+var SubjectPublicKeyInfo = asn1.define('SubjectPublicKeyInfo', function() {
+  this.seq().obj(
+    this.key('algorithm').use(AlgorithmIdentifier),
+    this.key('subjectPublicKey').bitstr()
+  );
+});
+rfc5280.SubjectPublicKeyInfo = SubjectPublicKeyInfo;
+
+// TBSCertList  ::=  SEQUENCE  {
+//      version                 Version OPTIONAL,
+//      signature               AlgorithmIdentifier,
+//      issuer                  Name,
+//      thisUpdate              Time,
+//      nextUpdate              Time OPTIONAL,
+//      revokedCertificates     SEQUENCE OF SEQUENCE  {
+//           userCertificate         CertificateSerialNumber,
+//           revocationDate          Time,
+//           crlEntryExtensions      Extensions OPTIONAL
+//      }  OPTIONAL,
+//      crlExtensions           [0] Extensions OPTIONAL }
+var TBSCertList = asn1.define('TBSCertList', function() {
+  this.seq().obj(
+    this.key('version').optional().int(),
+    this.key('signature').use(AlgorithmIdentifier),
+    this.key('issuer').use(Name),
+    this.key('thisUpdate').use(Time),
+    this.key('nextUpdate').use(Time),
+    this.key('revokedCertificates').optional().seq().obj(
+      this.seq().obj(
+        this.key('userCertificate').int(),
+        this.key('revocationDate').use(Time),
+        this.key('crlEntryExtensions').optional().seqof(Extension)
+      )
+    ),
+    this.key('crlExtensions').implicit(0).optional().seqof(Extension)
+  );
+});
+rfc5280.TBSCertList = TBSCertList;
+
+// Extension  ::=  SEQUENCE  {
+//      extnID      OBJECT IDENTIFIER,
+//      critical    BOOLEAN DEFAULT FALSE,
+//      extnValue   OCTET STRING }
+var Extension = asn1.define('Extension', function() {
+  this.seq().obj(
+    this.key('extnID').objid(x509OIDs),
+    this.key('critical').bool().def(false),
+    this.key('extnValue').octstr().contains(function (obj) {
```

Comment:
No need in space between `function` and `(`, it is not consistent with the rest of the file.